app/components/Form/MultiSelectGroup.tsx

```diff
@@ -27,13 +27,23 @@ const MultiSelectGroup = ({ name, label, children }: Props) => {
         subscription={{ errors: true, submitErrors: true, touched: true }}
       >
         {(props) => {
-          const error = props.errors ? props.errors[name] : [];
+          let error = '';
+          if (Array.isArray(props.errors?.[name])) {
+            error = Object.values(props.errors[name][0]).join('');
+          } else if (
+            props.errors?.[name] &&
+            typeof props.errors[name] === 'string'
+          ) {
+            error = props.errors[name];
+          }
+
```

Comment:
This works, but kinda annoying that there is no obvious way to prevent it from sometimes returning the error as an array and sometimes as a string.

Reply:
Yup, I will look further into it though. In the meantime we need a working solution